From b661c34ac60daeb13f8fd52ee06f5101583334cd Mon Sep 17 00:00:00 2001
From: Dan McDonald <danmcd@joyent.com>
Date: Thu, 16 Aug 2018 14:08:51 -0400
Subject: [PATCH] OS-7141 Overlay device-creation weirdness contributes to
 varpd boot failures Reviewed by: Robert Mustacchi <rm@joyent.com> Reviewed
 by: Jason King <jbk@joyent.com>

---
 overlay/generic/etc/name_to_major | 1 +
 1 file changed, 1 insertion(+)

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index cfab8bff..26eaeddb 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -193,6 +193,7 @@ ixgbe 192
 bpf 193
 lsimega 194
 vioif 195
+overlay 196
 mpt_sas 198
 mr_sas 199
 mega_sas 200
-- 
2.21.0

